.gitattributes

```diff
@@ -0,0 +1 @@
+.git_archival.txt  export-subst
```

Comment:
@javierggt does this need newline or fine without it?

Reply:
I don't know for sure, but I believe it is not necessary

Reply:
OK.  @jzuhone I don't have a quick way to test that so I think it is fine as well; if we end up with any weird things when we build packages we can fix later.